src/citrine/informatics/design_spaces/formulation_design_space.py

```diff
@@ -37,26 +36,23 @@ class FormulationDesignSpace(Resource['FormulationDesignSpace'], DesignSpace, AI
 
     """
 
-    _resource_type = ResourceTypeEnum.MODULE
-
     formulation_descriptor = properties.Object(
         FormulationDescriptor,
-        'config.formulation_descriptor'
+        'data.instance.formulation_descriptor'
     )
-    ingredients = properties.Set(properties.String, 'config.ingredients')
+    ingredients = properties.Set(properties.String, 'data.instance.ingredients')
```

Comment:
Where in the backend does this correspond to? `data.instance` -> `OrionDesignSpace` (Formulation in this case)?

Reply:
Correct. If it's a top-level DS, then it's just the new structure. If it's a subspace of a product DS, then when the product DS is deserialized it wraps its children in the scaffolding of the top-level object so the polymorphic serializable can do its thing.